app-api/src/main/java/org/openmrs/eip/app/management/entity/SenderSyncResponse.java

```diff
@@ -0,0 +1,49 @@
+package org.openmrs.eip.app.management.entity;
+
+import java.util.Date;
+
+import javax.persistence.Access;
+import javax.persistence.AccessType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Table;
+import javax.validation.constraints.NotNull;
+
+@Entity
+@Table(name = "sender_sync_response")
+public class SenderSyncResponse extends AbstractEntity {
+	
+	public static final long serialVersionUID = 1;
+	
+	@NotNull
+	@Column(name = "message_uuid", length = 38, nullable = false, updatable = false)
+	@Access(AccessType.FIELD)
+	private String messageUuid;
```

Comment:
Why is the access type set to field? This field has a public setter, I think you can remove the @Access annotation

Reply:
This was put there when I was using a constructor to set the messageUuid once, and at that time the setter did not exist.
But then you told me to remove the constructor and this annotation was left by mistake after creating the setter. I will remove it.